data/seeds/010-meetings.js

```diff
@@ -14,7 +14,7 @@ const dummyData = [
     host_id: '7',
     attendee_id: '00ultx74kMUmEW8054x6',
     meeting_notes: 'Remember to bring a smile',
-    meeting_missed: 'Missed'
+    meeting_missed: 'Missed',
```

Comment:
re our conversation: this is the file I'm referring to. See the trailing comma added and the `host_id` is a str? We can let this ride though because it's not work that pertains to your ticket.

Reply:
Yes, the trailing comma was added by me since I saw the eslint error and I couldn't let it slide but the host_id string needs to be fixed. I will see what I can do to fix that.